src/components/OcSelect.vue

```diff
@@ -1,27 +1,36 @@
 <template>
-  <vue-select
-    ref="select"
-    :disabled="disabled"
-    :filter="filter"
-    class="oc-select"
-    v-bind="$attrs"
-    v-on="$listeners"
-  >
-    <template #search="{ attributes, events }">
-      <input class="vs__search" v-bind="attributes" @input="userInput" v-on="events" />
-    </template>
-    <template v-for="(index, name) in $scopedSlots" #[name]="data">
-      <slot v-if="name !== 'search'" :name="name" v-bind="data" />
-    </template>
-    <div slot="no-options" v-translate>No options available.</div>
-    <template #spinner="{ loading }">
-      <oc-spinner v-if="loading" />
-    </template>
-  </vue-select>
+  <div>
+    <label :for="id" v-text="label" />
```

Comment:
```suggestion
    <label :for="id" class="oc-label" v-text="label" />
```

Just checked how the update journey in `web` would look like and we seem to give all labels this class, which doesn't style anything initially but sometimes is used to target the label for styling

Reply:
Makes sense but possibly belongs into the original pr

Reply:
True, but that one is merged already (and I somehow ended here without understanding this _is not the original_ ;D my bad, could you include it either way?